packages/runtime/src/slate/ListPlugin/ListPlugin.ts

```diff
@@ -90,6 +90,7 @@ export function onKeyDown(e: KeyboardEvent, editor: Editor) {
 }
```

Comment:
Missed this before. More to come in this commit

Reply:
Missed what, exactly?

Reply:
When I dropped the list plugin we only had one set of normalization so not calling `normalizeNode(entry` at the end wasn't an issue. For the normalization to cascade through all normalization plugins from the different plugins I have to call `normalizeNode` at the end like this diff changes.

> More to come in this commit
Refers to all the changes that I am going to make with the List plugin to get it to match the new plugins